monitoring/executor.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+func ExecutorAndExecutorQueue() *Container {
+	return &Container{
+		Name:        "executor",
+		Title:       "Executor And Executor Queue",
```

Comment:
I notice you use `executor-queue` in `sharedContainerCPUUsage` - is this a single service, and is the service called `executor-queue`? note that our alerts integration expects the dashboard name to be the same as the service name:

https://github.com/sourcegraph/sourcegraph/blob/main/docker-images/prometheus/cmd/prom-wrapper/receivers.go#L59

in which case:

```suggestion
		Name:        "executor-queue",
		Title:       "Executor Queue",
```

Reply:
Yeah this dashboard is a little strange because `executor` runs outside of the deployment. I'm going to make this the official `executor-queue` dashboard (the service that actually is deployed with the instance) and then tack some of the executor metrics on this one.

Might end up splitting them out later, but this is from 0-1 so it's a infinity% improvement.